ecosystem/runners/runner.py

```diff
@@ -43,6 +47,39 @@ def tear_down(self):
         if self.cloned_repo_directory and os.path.exists(self.cloned_repo_directory):
             shutil.rmtree(self.cloned_repo_directory)
 
+    def get_config(self, file, file_fault, ecosystem_deps: Optional = None):
+        """check for configuration file"""
```

Comment:
Add rich docstring to explain what is going on here and describe parameters in google docstring format
Something like this :) 

https://github.com/qiskit-community/ecosystem/blob/21f1e80d72520af1683457ea14169481713bdbd8/ecosystem/daos/jsondao.py#L45-L53